### PR TITLE
Df 1457 events page

### PR DESCRIPTION
**Pull request by jimmynotjim**
Completes the Events landing page content (outside the events list)

## Additions

- None

## Removals

- None

## Changes

- Updated the events vars to temporarily use the blog query (necessary for subscribe render)
- Added Page header with dummy sub-head text
- Updated Subscribe, RSS and Related Links blocks for correct layout and formatting

## Testing

- navigate to /events

## Review

- @sebworks 
- @anselmbradford 
- @duelj 

## Preview

upcoming events cfgov refresh demo

## Notes

- The existing "Related Links" template uses arrows to the right of the text. If we want to remove these they should be removed globally
- Some pages pull related links from the View CPT and some have them hard coded. These could be updated to pull from the View when it's complete

### CODE REVIEW

File: events/index.html [hunk 1]
```diff
@@ -41,39 +41,55 @@
 
 {% block content_main %}
 
-    {# TEMP The mocked event content is in the following paragraph #}
-    <p>
-        Events content goes here: {{vars.mock_post.content}}
-    </p>
+    <section class="media block block__flush-top block__sub">
+        <div class="media_body">
+            <h1>Upcoming events</h1>
```
Comment (anselmbradford) [inline events/index.html]: Not to hold each of these up b/c of semantics, but you might consider moving heading content outside of a `<div>` with a name of `media_body`, `<h1>Upcoming events</h1>` will become the title for  `<section class="media block block__flush-top block__sub">` in the document outline.

**Comment by anselmbradford** [comment]
:+1: LGTM overall

File: events/index.html (hunk 1 above)
Comment (jimmynotjim) [inline events/index.html]: This is another existing pattern throughout the site. Worth considering but outside the scope of this PR.

Comment (anselmbradford) [inline events/index.html]: SG

**Pull request merged**